octosql/run.go

```diff
@@ -0,0 +1,143 @@
+package octosql
+
+import (
+	"context"
+	"fmt"
```

Comment:
Please use goimports with grouping to format this (this can be easily configured if you use goland).

Reply:
I changed the configuration. It should be fine now, but please let me know if the style still do not follow the project guidelines.